content/en/docs/challenge-types.md

```diff
@@ -120,7 +120,7 @@ Cons:
 
 This challenge was defined in draft versions of ACME. It did a TLS
 handshake on port 443 and sent a specific [SNI] header, looking for
-certificate that contained the token. It [will be disabled in March
+certificate that contained the token. It [was be disabled in March
```

Comment:
This says "It was be disabled..." - need to get rid of the word "be"

Reply:
🤦‍♂ 🤦‍♂ 🤦‍♂ 🤦‍♂ 🤦‍♂